<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="callDstEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>abc123</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clientChangeBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>更改</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>呼叫目标</string>
       </property>
      </widget>
      <widget class="QPushButton" name="hangupBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>挂断</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>客户端Id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="clientEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>abc456</string>
       </property>
      </widget>
      <widget class="QPushButton" name="callBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>呼叫</string>
       </property>
      </widget>
      <widget class="QPushButton" name="localShowBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>本地画面</string>
       </property>
      </widget>
      <widget class="QPushButton" name="remoteShowBtn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>远程画面</string>
       </property>
      </widget>
      <widget class="QPushButton" name="newLocalVideoBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>新建本地视频</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="videoNameEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>local_aaa</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removeLocalVideoBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>删除本地视频</string>
       </property>
      </widget>
      <widget class="QPushButton" name="msgSendBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>发送消息</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
